strings/cas_white.txt

```diff
@@ -1,10 +1,11 @@
 Those motherfucking carp.
 Having sex in the maintenance tunnels.
 Space 'Nam.
+Space capitalism.
 Space lesbians.
 Space NEETs.
 Space Hitler.
-Space Asshole.
+Space assholes.
```

Comment:
boooooo

Reply:
To clarify, I dislike this change and think that Space Asshole is better than genericising it

Reply:
Space Asshole fits into fewer black prompts, is the thing